lib/util/module-resolver.js

```diff
@@ -35,13 +35,10 @@ const DEFAULT_OPTIONS = {
  *      lookup with the highest priority paths coming first.
  * @constructor
  */
-function ModuleResolver(options) {
-    options = options || {};
-
-    this.options = Object.assign({}, DEFAULT_OPTIONS, options);
-}
-
-ModuleResolver.prototype = {
+class ModuleResolver {
```

Comment:
Should we be updating these JSDoc blocks? Looks like the class should have a description and then the above should be moved the constructor function JSDoc comment.

See: http://usejsdoc.org/howto-es2015-classes.html